src/languages/en-US/commands/admin.json

```diff
@@ -1,5 +1,4 @@
 {
-	"confGuarded": "{{name, toTitleCase}} may not be disabled.",
```

Comment:
This is unused and a remnant?

Reply:
I removed based on reference count on the `LanguageKeys`'s exports (with some exceptions being Sapphire's keys), and that one... just didn't show up at all.

Reply:
Upon further inspection, I saw that it's actually used, it just has no message:

https://github.com/skyra-project/skyra/blob/4b9d5cdb9fb66c8ea397def042984fdb714fffe1/src/lib/structures/preconditions/PermissionsPrecondition.ts#L47